src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -48,4 +49,23 @@
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  /**
+   * Get historical price data (close, high, low, open, and volume) for the given symbol
+   * over a specified time range. See https://iexcloud.io/docs/api/#historical-prices.
+   * @param symbol A string representing a stock symbol for which to retrieve historical data.
+   * @param range A string specifying a range of time. See link.
+   * @param date A string representing a date in YYYYMMDD format. See link.
+   * @return A list of IexHistoricalPrice objects for the symbol for each date in the range of time.
+   */
+
+  @GetMapping(value = "${mvc.iex.getHistoricalPricesPath}", produces = {
+      MediaType.APPLICATION_JSON_VALUE})
+  public List<IexHistoricalPrice> getHistoricalPrices(
+      @RequestParam(value = "symbol") final String symbol,
+      @RequestParam(value = "range", defaultValue = "1m") final String range,
+      @RequestParam(value = "date", defaultValue = "") final String date) {
```

Comment:
Default values are fine, but we should make sure that the requirements specified that. Additionally, we should add that to our Javadoc

Reply:
Added mention of default value to javadoc